### ISSUE TITLE
for destination local path - full path is required, or you get stepped folder creation

### ISSUE THREAD

**Issue by mohclips** [cooperspencer/gickup#30]
```
./gickup --version
v0.9.4
```

config.yml
```
destination:
   local:
     - path: ./backup
```

leads to

```
backup
  repo1
  backup
     repo2
     backup
        repo3
        backup
          repo4
          backup
```
and so on

**Comment by cooperspencer** [comment]
Hi,
thanks for bringing this to my attention. I will implement a fix for this issue.

**Comment by cooperspencer** [comment]
that should be fixed with 4d1b9006a3563aec9cd887e6fa22449bae58e600.
I will have a look on the other issue and create a new release after fixing that issue.

**Comment by cooperspencer** [comment]
Pushed a new release with the fixes.
